src/js/flows/search/mod.ts

```diff
@@ -5,14 +5,17 @@ import {handle} from "./handler"
 import Handlers from "../../state/Handlers"
 import {SearchResponse} from "./response"
 import {Handler} from "src/js/state/Handlers/types"
+import Current from "../../state/Current"
+import Tab from "../../state/Tab"
+import randomHash from "../../brim/randomHash"
 
 type Args = {
   query: string
-  from: Date
-  to: Date
-  spaceId: string
-  id: string
-  target: SearchTarget
+  from?: Date
+  to?: Date
+  spaceId?: string
+  id?: string
+  target?: SearchTarget
```

Comment:
I just added sensible defaults for all these fields so that we don't have to pass them in every time. 

from => whatever the start time of the current tab is
to => the end time of the current tab
spaceId => the space id of the current tab
id => a random hash (this is used to cancel a request if needed)
target => "events" instead of "archive".